src/sql/src/session/vars/definitions.rs

```diff
@@ -493,7 +493,7 @@ pub static MAX_REPLICAS_PER_CLUSTER: VarDefinition = VarDefinition::new(
 
 pub static MAX_CREDIT_CONSUMPTION_RATE: VarDefinition = VarDefinition::new_lazy(
     "max_credit_consumption_rate",
-    lazy_value!(Numeric; || 1024.into()),
+    lazy_value!(Numeric; || 24.into()),
```

Comment:
I'm a little concerned this will break existing environments. Looking at LaunchDarkly everything seems to be configured correctly, but this does open the risk for if LaunchDarkly fails to sync we'll now "fail closed" and prevent users from creating new objects, whereas before we'd "fail open"

Reply:
yeah, this probably doesn't actually need to change since we check the license key first for the value anyway